addon/components/plaid-text/template.hbs

```diff
@@ -0,0 +1,5 @@
+{{#if hasBlock}}
+  {{yield}}
```

Comment:
I don't have a particular use case in mind, but it seems like a nice option
to have.
On Jun 21, 2016 7:32 AM, "Ivan Vanderbyl" <notifications@github.com> wrote:

> In addon/components/plaid-text/template.hbs
> <https://github.com/ivanvanderbyl/maximum-plaid/pull/23#discussion_r67858716>
> :
>
> > @@ -0,0 +1,5 @@
> > +{{#if hasBlock}}
> > +  {{yield}}
>
> What's the use case for needing to call this with a block?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ivanvanderbyl/maximum-plaid/pull/23/files/b9d12de243a37dfa3f31f1976133e07ff65904f1#r67858716>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AApQf_FY9D5mvAAm0XnxNSUD508p1uoqks5qN9n3gaJpZM4I6MXO>
> .
>